client/utils/auth.py

```diff
@@ -24,7 +24,18 @@
 # The client secret in an installed application isn't a secret.
 # See: https://developers.google.com/accounts/docs/OAuth2InstalledApp
 CLIENT_SECRET = 'zGY9okExIBnompFTWcBmOZo4'
-REFRESH_FILE = '.ok_refresh'
+
+
+def get_config_directory():
+    return os.path.expanduser('~') + '/.config/ok/'
```

Comment:
Since this path is a constant throughout one OK session, I suggest making it a global variable.

Also, use `os.path.join(..., '.config', 'ok')`, in case someone is using a native Windows terminal (path delimiters would be backslashes).